content/applications/sales/crm/track_leads/resellers.rst

```diff
@@ -0,0 +1,142 @@
+   individual record overwrites the default weight assigned on the level configuration form.
+
+.. _crm/partner-activations:
+
+Partner activations
+===================
+
+Partner *activations* are used to identify the status of a partner. Activations are assigned on an
+individual contact record, and can be used to group or filter the *Partnership Analysis* report
+(:menuselection:`CRM app --> Reporting --> Partnerships`).
+
+To view the partner levels, navigate to :menuselection:`CRM app --> Configuration --> Partner
+Activations`.
+
+Three activation types are created by default in the *CRM* app:
+
+- :guilabel:`Fully Operational`
+- :guilabel:`Ramp-up`
+- :guilabel:`First Contact`
+
+New partner activations can be added, as needed, by clicking :guilabel:`New`, and entering a
+:guilabel:`Name` on the new line that appears. Then, select the desired status in the
+:guilabel:`Active` column.
+
+Existing partner activations can also be edited and renamed, if desired. To rename a status, click
+the :guilabel:`Name` field of a desired level, and enter a new name.
+
+To change the active status of an activation, slide the toggle in the :guilabel:`Active` column of
+the desired activation to the *inactive* position.
+
+.. figure:: resellers/activations-toggle.png
+   :align: center
+   :alt: The list of default partner activations in the CRM app.
+
+   The list of default Partner Activations in the CRM app. The toggle for First Contact is in the
+   inactive position, while the rest are active.
+
+Partner assignments
+===================
+
+After partner levels and partner activations have been configured, individual partner records can be
+configured.
+
+To update an individual partner record, navigate to :menuselection:`CRM app --> Sales -->
+Customers`, and click the Kanban card for the desired partner to open the customer record.
+
+On the customer record, click the :guilabel:`Partner Assignment` tab.
+
+Click the :guilabel:`Partner Level` field, and select an option from the drop-down menu to assign a
+level. Click the :guilabel:`Activation` field, and select a partner activation type from the
+drop-down list, if desired. Then, click the :guilabel:`Level Weight` field to assign a different
+level weight, if necessary.
+
+Publish partners
+================
+
+With the Odoo *Website* app installed, a new webpage can be created to display a list of all active
+partners from the *CRM* app. Navigate to :menuselection:`Website app --> Site --> Pages`, and click
+:guilabel:`New`. Hover over the :guilabel:`Blank Page` template, and click the :guilabel:`Use this
+template` button to create a new page.
+
+.. image:: resellers/blank-page-template.png
+   :align: center
+   :alt: The create new page pop-up in the Website app.
+
+In the :guilabel:`New Page` pop-up window, enter a :guilabel:`Page Title` (e.g. `Partners`). Then,
+click :guilabel:`Create`. In the toolbar on the far-right of the page, click :guilabel:`Save`.
```

Comment:
This doesnt seem correct to me, and maybe I am missing something– but please double check the following:

The `<yourodoowebsite.com>/partners` page is created automatically. Any published partners automatically display on this page, grouped by their partner level. I don't see a need to create a "Partners" page, nor (do people say _nor_? 🤔) do the instructions detail how to add the dynamic partners to the new page for display.

I am also pretty sure **only** partners that have a contact type as _Company_ are displayed on this website page, which may be a good idea to specify.

```suggestion
With the Odoo *Website* app installed, a new webpage is created to display a list of all active
partners from the *CRM* app.
```